mlflow/store/artifact/hdfs_artifact_repo.py

```diff
@@ -149,7 +149,11 @@ def download_artifacts(self, artifact_path, dst_path=None):
             return local_dir
 
     def _download_file(self, remote_file_path, local_path):
```

Comment:
@shidianshifen Thanks for the contribution! Can we add a test?

Reply:
thx for reply, test is added

Reply:
Thanks! Have you manually tested this patch works?

Reply:
yes, i tested in our self-host env